foam_controlwrapper/examples/poiseuille.py

```diff
@@ -3,12 +3,12 @@
 """
 
 from simphony.core.cuba import CUBA
```

Comment:
@jmarcelogimenez , this comment has been not addressed.  An issue should be opened if it is not going to be addressed in this PR.